result/bgpprefix.yaml

```diff
@@ -16,6 +16,53 @@ components:
           x-constraint:
             - "/components/schemas/Bgp.v4Peer/properties/name"
             - "/components/schemas/Bgp.v6Peer/properties/name"
+        ipv4_unicast_filters:
```

Comment:
How the user will specify that he is interested only in ipv4_unicast or ipv6_unicast prefixes? If "ipv4_unicast_filters" is empty or null, what will be the interpretation - return all or nothing? We need to cover the following:
(a) return all prefixes ==> Assuming no filter should support this scenario
(b) return all ipv4_unicast prefixes only or ipv6_unicast prefixes only ==> if this is supported?
(c) return specific ipv4/v6 unicast prefix ==> this is supported with the filter.

Reply:
https://github.com/open-traffic-generator/models/pull/244/commits/24f9d8ecc5963bef4c07edd6c4b4cc022a7837dd